libdeno/api.cc

```diff
@@ -123,6 +126,19 @@ int deno_execute(Deno* d_, void* user_data, const char* js_filename,
   return deno::Execute(context, js_filename, js_source) ? 1 : 0;
 }
 
+int deno_execute_mod(Deno* d_, void* user_data, const char* js_filename,
+                     const char* js_source) {
+  auto* d = unwrap(d_);
+  deno::UserDataScope user_data_scope(d, user_data);
+  auto* isolate = d->isolate_;
+  v8::Locker locker(isolate);
+  v8::Isolate::Scope isolate_scope(isolate);
+  v8::HandleScope handle_scope(isolate);
+  auto context = d->context_.Get(d->isolate_);
+  CHECK(!context.IsEmpty());
+  return deno::ExecuteMod(context, js_filename, js_source) ? 1 : 0;
```

Comment:
What's this return value? Success/error? In c++ code I would expect to use bool, or alternatively you can do common c way of 0 (ok) / -1 (fail).

Reply:
See docs in deno.h - this is to match the behavior of deno_execute  1 succuess / 0 fail.

(Yes I agree it should be the opposite, but let's do that in a separate PR.)